Sources/Responses/EventItemType.swift

```diff
@@ -35,20 +37,36 @@ public class EventItem: BoxModel {
     /// - Throws: Decoding error.
     public required init(json: [String: Any]) throws {
         rawData = json
+        var updatedJson: [String: Any] = json
 
-        guard let type = json["type"] as? String else {
+        if let entry = updatedJson.removeValue(forKey: "item_id") {
```

Comment:
Just curious, why are we removing each of the keys and adding them back to updatedJson? Are we changing the keys to map to their respective correct key entries?
```
item_id->id
item_type->type
item_name->name
```

Reply:
The events endpoint returns objects with `item_id`, `item_type`, and `item_name` as keys. If we use these keys, the deserialization to folders, files, users, etc. fails.